docs/index.rst

```diff
@@ -2104,6 +2104,71 @@ You will need to configure manually Foundry. Unfortunately, this may mean duplic
 
 .. _stories:
 
+In-memory Behavior
+~~~~~~~~~~~~~~~~~~
+
+Foundry allows to use "in-memory" repositories in your factories. This is mainly useful for `DDD <https://en.wikipedia.org/wiki/Domain-driven_design>`_
+applications or with `hexagonal architecture <https://en.wikipedia.org/wiki/Hexagonal_architecture_(software)>`, where
```

Comment:
It seems like this link is broken in the rendered docs: 

[![grafik](https://github.com/user-attachments/assets/56bb12cf-8189-4b80-a739-d731e2fda9a1)](https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#in-memory-behavior)

Reply:
Hi @jdreesen 

thanks for reporting! I've [already fixed it](https://github.com/zenstruck/foundry/pull/904) but the docs are only generated once per day